fe/fe-core/src/main/java/com/starrocks/catalog/Database.java

```diff
@@ -436,6 +440,31 @@ public boolean createTableWithLock(Table table, boolean isReplay) {
                     // Write edit log
                     CreateTableInfo info = new CreateTableInfo(fullQualifiedName, table);
                     GlobalStateMgr.getCurrentState().getEditLog().logCreateTable(info);
+                    if (table.isOlapTable()) {
+                        OlapTable olaptable = (OlapTable) table;
+                        if (!olaptable.isBinlogConfigNull()) {
+                            UpdateBinlogConfigInfo updateBinlogConfigInfo = new UpdateBinlogConfigInfo(id,
+                                    table.getId(), olaptable.getCurBinlogConfig(), null);
+                            GlobalStateMgr.getCurrentState().getEditLog().logModifyBinlogConfig(updateBinlogConfigInfo);
```

Comment:
What if `logCreateTable` is successful, but `logModifyBinlogConfig` fail? Or maybe we can `logModifyBinlogConfig` first, and discard the log if not find the table when replaying the binlog config log?